support-frontend/assets/components/countryGroupSwitcher/countryGroupSwitcher.scss

```diff
@@ -36,9 +36,13 @@
 		.svg-dropdown-arrow path {
 			fill: gu-colour(highlight-main);
 		}
+    .svg-dropdown-arrow{
+      transform: rotate(180deg);
+    }
 	}
 
 	.component-select-input__label {
-		@include visually-hidden;
+		//@include visually-hidden;
 	}
```

Comment:
Is this meant to be included? If so, would it be possible to delete it rather than comment it out?

Reply:
I have made a change to the file to align the dropDown Arrow  with the link